gocse/csfle/data_key.go

```diff
@@ -30,14 +33,28 @@ func GetDataKey(keyVaultNamespace, uri string, provider kms.Provider) (string, e
 		_ = clientEnc.Close(context.TODO())
 	}()
 
-	// specify the master key information that will be used to
-	// encrypt the data key(s) that will in turn be used to encrypt
-	// fields, and create the data key
-	dataKeyOpts := options.DataKey().SetMasterKey(provider.DataKeyOpts())
-	dataKeyID, err := clientEnc.CreateDataKey(context.TODO(), provider.Name(), dataKeyOpts)
+	// look for a data key
+	keyVault := strings.Split(keyVaultNamespace, ".")
+	db := keyVault[0]
+	col := keyVault[1]
```

Comment:
Should we use "coll" instead of "col" since that seems more commonly used as a variable name in server documentation? My guess is it can help distinguish it from "col" which is frequently used to abbreviate "column".